src/sourmash/sourmash_args.py

```diff
@@ -391,14 +391,22 @@ def load_pathlist_from_file(filename):
     try:
         with open(filename, 'rt') as fp:
             file_list = [ x.rstrip('\r\n') for x in fp ]
-
-        if not os.path.exists(file_list[0]):
-            raise ValueError("first element of list-of-files does not exist")
+        if len(file_list) == 0:
```

Comment:
note, this can be abbreviated more Pythonically as:
```
if not file list:
...
```
(in Python, empty lists evaluate to False)

Reply:
Okay, I'll fix that!

Reply:
bump!

Reply:
Please revisit this comment, thanks - https://github.com/dib-lab/sourmash/pull/1469/files#r624171173